demos/Makefile

```diff
@@ -1,5 +1,5 @@
-cases := base_case 2d_compressible_ALA 2d_compressible_TALA viscoplastic_case 2d_cylindrical 3d_spherical 3d_cartesian adjoint multi_material gplates_global free_surface
-demo_cases := base_case 2d_cylindrical 3d_spherical 2d_compressible_ALA visualise_ALA_p_nullspace 2d_compressible_TALA viscoplastic_case 3d_cartesian gplates_global multi_material/compositional_buoyancy multi_material/thermochemical_buoyancy adjoint free_surface
+cases := base_case 2d_compressible_ALA 2d_compressible_TALA viscoplastic_case 2d_cylindrical 3d_spherical 3d_cartesian adjoint multi_material gplates_global free_surface viscoelastic
+demo_cases := base_case 2d_cylindrical 3d_spherical 2d_compressible_ALA visualise_ALA_p_nullspace 2d_compressible_TALA viscoplastic_case 3d_cartesian gplates_global multi_material/compositional_buoyancy multi_material/thermochemical_buoyancy adjoint free_surface viscoelastic
```

Comment:
For ease of navigation, as the number of demos and tests grow, I'd say start a viscoelastic folder, and mark this as base. I think it's probably now an idea to also add a mantle_dynamics folder, throwing all other demos in there, excluding multi_material, which may justify it's own folder.

Reply:
I'll have a go at doing this!